wagtail/models/__init__.py

```diff
@@ -317,14 +318,15 @@ class Page(AbstractPage, index.Indexed, ClusterableModel, metaclass=PageBase):
         verbose_name=_("latest revision created at"), null=True, editable=False
     )
     live_revision = models.ForeignKey(
-        "Revision",
+        "wagtailcore.Revision",
         related_name="+",
         verbose_name=_("live revision"),
         on_delete=models.SET_NULL,
         null=True,
         blank=True,
         editable=False,
     )
+    _revisions = GenericRelation("wagtailcore.Revision", related_query_name="page")
```

Comment:
Should this use a non-"private" name?

Reply:
This looks OK to me, I don't think we'd ever want someone to call this directly?